### PR TITLE
Cleanup: remove duplicated resource models

### PR DESCRIPTION
**Pull request by alfonsomthd**
- Keep `CephClusterModel` & `OCSStorageCLusterModel` in `shared` library as the library is being used by other projects.

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
Hope we may need to update the test case also ...

File: packages/ocs/hooks/useOcsHealth.ts [hunk 1]
```diff
@@ -13,11 +14,7 @@ import {
   WatchK8sResource,
 } from '@openshift-console/dynamic-plugin-sdk';
 import * as _ from 'lodash-es';
-import {
-  CephClusterModel,
-  CephObjectStoreModel,
-  NooBaaSystemModel,
-} from '../models/core';
+import { CephObjectStoreModel, NooBaaSystemModel } from '../models/core';
```
Comment (GowthamShanmugam) [inline packages/ocs/hooks/useOcsHealth.ts:17]: is that ok to keep some OCS and ODF models under non-shared repo? In future other plugins also may use these types right? What about moving all ODF/OCS related types to shared?

File: packages/ocs/storage-pool/body.tsx [hunk 2]
```diff
@@ -118,7 +118,11 @@ export const StoragePoolBody: React.FC<StoragePoolBodyProps> = ({
   const poolNs = getNamespace(cephCluster);
 
   const [storageCluster, storageClusterLoaded, storageClusterLoadError] =
-    useK8sGet<ListKind<StorageClusterKind>>(StorageClusterModel, null, poolNs);
+    useK8sGet<ListKind<StorageClusterKind>>(
```
Comment (GowthamShanmugam) [inline packages/ocs/storage-pool/body.tsx:121]:
```suggestion
    useK8sGet<ListKind<OCSStorageClusterKind>>(
```

Optional comment

File: packages/ocs/hooks/useOcsHealth.ts (hunk 1 above)
Comment (alfonsomthd) [reply packages/ocs/hooks/useOcsHealth.ts:17]: It's all about team consensus: if we agree on moving ODF/OCS types & models to `shared` then we can create a technical issue to address that in another PR: here I'm only removing duplicates that add confusion when deciding where to import from.

File: packages/ocs/storage-pool/body.tsx (hunk 2 above)
Comment (alfonsomthd) [reply packages/ocs/storage-pool/body.tsx:121]: @GowthamShanmugam Although I understand this suggestion for naming consistency, it'll have side effects (breaking changes) in other projects (when there is an upgrade of the `shared` library version).
Waiting for @SanjalKatiyar feedback in order to procced or not with this suggestion.

Comment (SanjalKatiyar) [reply packages/ocs/storage-pool/body.tsx:121]: we don't use `OCSStorageClusterModel` or `StorageClusterKind` in Fusion, only place it's used in this repo...

we can either remove `OCS` part from `OCSStorageClusterModel`, or add `OCS` to `StorageClusterKind`... either is alright...

File: packages/ocs/hooks/useOcsHealth.ts (hunk 1 above)
Comment (GowthamShanmugam) [reply packages/ocs/hooks/useOcsHealth.ts:17]: ack, it could be a separate topic to discuss

File: packages/ocs/storage-pool/body.tsx (hunk 2 above)
Comment (alfonsomthd) [reply packages/ocs/storage-pool/body.tsx:121]: @GowthamShanmugam @SanjalKatiyar then I'll remove `OCS` prefix so the name it's more aligned with the CR (`StorageCluster`).

**Pull request merged**